app/controllers/accountants_controller.rb

```diff
@@ -5,28 +5,29 @@ class AccountantsController < ApplicationController
   before_action :find_patient, only: [:edit]
 
   def index
-    patients = pledged_patients.includes(:clinic)
-    @patients = paginate_results(patients)
+    @patients = paginate_results(pledged_patients)
+    @entry_name = t('common.patient').downcase
   end
 
   def search
     have_search = params[:search].present?
     have_clinic = params[:clinic_id].present?
 
-    if have_search || have_clinic
-      partial = Patient.where(pledge_sent: true)
-                       .includes(:clinic)
-                       .includes(:fulfillment)
-                       .order(pledge_sent_at: :desc)
+    @results = if have_search || have_clinic
+      partial = pledged_patients
 
       partial = partial.where(clinic_id: params[:clinic_id]) if have_clinic
-      partial = partial.search(params[:search]) if have_search
+      partial = partial.search(params[:search], search_limit: nil) if have_search
 
-      @results = partial
+      partial
     else
-      @results = pledged_patients
+      pledged_patients
     end
 
+    # when we search, counter says 'results' instead of 'patients'
+    @entry_name = t('accountants.results')
+    @results = paginate_results @results
```

Comment:
Rails convention nit: Usually we want to avoid using instance variables that aren't directly returned by the controller, so it would be slightly better to have it be `results` until it's the version that we'll return